release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2059,6 +2059,31 @@ It is recommended that SR-IOV customers should not upgrade to {product-title} 4.
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930469[*BZ#1930469*])
 
+* In {product-title} 4.7, `configInformers` added to the Operator infrastructure code unsuccessfully start. As a result, the `ConfigObserver` fails to sync the cache. When this happens, the oVirt CSI Driver Operator shuts down after a couple of minutes, which leads to continual restarts. As a workaround, you can perform the following procedure:
+
+. Switch project to cluster with an oVirt CSI Operator:
++
+[source,terminal]
+----
+$ oc project openshift-cluster-csi-drivers
+----
+
+. Check for warnings:
++
+[source,terminal]
+----
+$ oc status
+----
+
+. Get pods:
++
+[source, terminal]
+----
+$ oc get pods
```

Comment:
It seems odd to me that the procedure stops here, but if getting the pods is enough, great!